release 1.1-8

pengdake opened the pull request:
1.[init][update] fix some problem about tools self update
2.[template] update nova profile for soft-delete
3.[manage][volume] support to delete private type volume
4.[manage][volume] support non-interactive mode to delete volume
5.[Bugfix][doctor][cls] print "checking haproxy resource"
6.添加 eayunstack cleanup orphan 功能
7.Fix redmine #6524
8.Fix issue #224
9.[bugfix]fix redmine #7317
10.fix redmine 7390
11.[feature]filter max_pool_size in confuguration files
12.Fix `eayunstack -h` run on fuel ERROR

Signed-off-by: pq 19921207pq@gmail.com

blkart commented:
**[属于1.1-7]** 1.[init][update] fix some problem about tools self update
**[属于1.1-7]** 2.[template] update nova profile for soft-delete
**[属于1.1-7]** 3.[manage][volume] support to delete private type volume
**[属于1.1-7]** 4.[manage][volume] support non-interactive mode to delete volume
**[属于1.1-7]** ]5.[Bugfix][doctor][cls] print "checking haproxy resource"
6.添加 eayunstack cleanup orphan 功能
7.Fix redmine #6524
8.Fix issue #224
9.[bugfix]fix redmine #7317
10.fix redmine 7390
11.[feature]filter max_pool_size in confuguration files  **redmine #7433**
12.Fix eayunstack -h run on fuel ERROR

---
- 不要把以前版本的更新内容写进来
- 有 redmine 编号的尽量把编号写上

blkart commented on eayunstack-tools.spec:
```diff
@@ -78,6 +78,9 @@ fi
 
 
 %changelog
+* Wed May 25 2016 blkart <blkart.org@gmail.com> 1.1-8
```
这一行有问题

pengdake replied:
最上面commit的日期不是最新的啊，所以我用的第二个，这个怎样处理啊

blkart commented on eayunstack-tools.spec:
```diff
@@ -78,6 +78,9 @@ fi
 
 %changelog
+* Wed May 25 2016 blkart <blkart.org@gmail.com> 1.1-8
+- commit 9f5c33bdb400d3a6ff3711d524027f5911164c0e 
```
不是 master 分支最后一个 commit id

pengdake replied:
同上

blkart commented on eayunstack-tools.spec:
```diff
@@ -78,6 +78,9 @@ fi
 
 %changelog
+* Wed Apr 20 2016 pq     <19921207pq@gmail.com> 1.1-8
+- commit 3c20e3f2ba8bd017f8f9147fc7ef4420a325be3d 
```
最后一个 commit id 不是 3da26e77ff8724eb22d35fd47723ca684d3afd8c 吗？

pengdake committed:
release 1.1-8

1.添加eayunstack cleanup orphan 功能
2.Fix redmine #6524
3.fix redmine 7390
4.[feature]filter max_pool_size in configuration files redmine #7433
5.Fix `eayunstack -h` run on fuel ERROR redmine #7317
6.[tools][doctor][env] checkmongodb service redmine #6359

Signed-off-by: pq <19921207pq@gmail.com>
